packages/logger/logger-commons/src/logger.ts

```diff
@@ -46,6 +44,8 @@ export function createLogger(
       req: pino.stdSerializers.req,
       res: pino.stdSerializers.res,
     },
+    sync: options.sync,
+    redact: options.redact,
```

Comment:
Great idea, additionally:
-  we might need to update docs
- update `yaml config files`  just like you did at the PR


https://verdaccio.org/docs/logger